static/app/components/replays/configureMobileReplayCard.tsx

```diff
@@ -16,7 +16,7 @@ function getPath(sdkName: string | null | undefined) {
     case 'sentry.java.android':
       return 'android'; // https://docs.sentry.io/platforms/android/session-replay/
     case 'sentry.dart.flutter':
-      return 'flutter'; // https://docs.sentry.io/platforms/flutter/session-replay/
+      return 'flutter'; // https://docs.sentry.io/platforms/dart/guides/flutter/session-replay/
```

Comment:
```suggestion
      return 'dart/guides/flutter'; // https://docs.sentry.io/platforms/dart/guides/flutter/session-replay/
```
the path should return the string needed to complete `...docs.sentry.io/platforms/${path}/session-replay/` used here

https://github.com/getsentry/sentry/blob/6c89ef33530add672966b67805e0d7cd5746d55f/static/app/components/replays/configureMobileReplayCard.tsx#L31-L49

Reply:
ah good catch! 🥔 thank you @michellewzhang